### ISSUE TITLE
not_contains should be "NOT LIKE", not "NOTLIKE"

### ISSUE THREAD

**Issue by alex-way** [rbw/pysnow#162]
The documentation on the [ServiceNow website](https://docs.servicenow.com/bundle/paris-platform-user-interface/page/use/common-ui-elements/reference/r_OpAvailableFiltersQueries.html) is conflicting. It suggests that the NOT LIKE operator shouldn't have a space in between. However, the example clearly shows a space between.

Testing in our production orlando instance the correct operator contains a space.

**Comment by vrsh** [comment]
As per ServiceNOW documentation and feature provided in not_contains method of pysnow, the correct reference is NOT LIKE
@alex-way If the Docstring of the method was your concern I have modified it.

**Comment by rbw** [comment]
@alex-way You're correct. The ServiceNow documentation is ambiguous about the *does not contain* operator; It suggests both *NOT LIKE* and *NOTLIKE*.

I think *NOTLIKE* would be more appropriate, as it'd be consistent with rest of the operator values. But ServiceNow decided to use *NOT LIKE* instead. This minor lack of consistency caused this *Issue*, and probably the documentation error - both in pysnow and the ServiceNow official docs.
That shows how important consistency is I guess :)

I checked the other library I'm working on, aiosnow, and there I used **!*** instead (maybe for this reason, can't remember).

https://aiosnow.readthedocs.io/en/latest/reference/fields/string.html#aiosnow.fields.string.StringQueryable.contains

Anyway, @vrsh is correct, the correct query operator value is *NOT LIKE*.
